tests/ftp-mimetype-detection/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  features:
+      - HAVE_MIMETYPE
```

Comment:
@jasonish seeing green CI for this highlights that we need SV coverage ;-)

Reply:
Please elaborate? What would this look like?

Reply:
You seemed to have more ideas than me.

I was just saying that manual runs should be done for releases to ensure that every SV test/check is run.
You were saying that there were tools to do that ;-)

Reply:
No tools that I know off.  I think Victor put out a wish directed my way that tools exists, to which I had no comment :)